.github/workflows/fuzz.yaml

```
@@ -0,0 +1,25 @@
+name: Fuzzing test
+
+on:
+  schedule:
+  - cron: 0 6 * * 6
+
+permissions:
+  contents: read  # for actions/checkout to fetch code
```

please change it to `read-all`, because it is unclear what permissions are in use for other types

```suggestion
permissions: read-all
```